users/talljoe/rules.mk

```
@@ -5,3 +5,5 @@ EXTRAFLAGS+=-flto
 TAP_DANCE_ENABLE=yes
 CONSOLE_ENABLE=no
 COMMAND_ENABLE=no
+DYNAMIC_KEYMAP_ENABLE=no
+BOOTMAGIC=yes
```

This should be `BOOTMAGIC_ENABLE`, and `yes` is supported, but it really is `full`, `lite` or `no` for the values now.

```suggestion
BOOTMAGIC_ENABLE=full
```

---

Thanks, that looks like it was leftover from me trying to troubleshoot why the Zeal60 updates broke my layout.